pkg/config/config_test.go

```diff
@@ -0,0 +1,66 @@
+package config_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/spf13/afero"
+
+	"code.cloudfoundry.org/quarks-utils/pkg/config"
+)
+
+var _ = Describe("Shared config", func() {
+	Describe("constructors", func() {
+		It("sets only meltdown defaults from constants, without manipulating arguments", func() {
+			for _, fs := range []afero.Fs{nil, afero.NewOsFs()} {
+				sharedConfig := config.NewDefaultConfig(fs)
```

Comment:
nit: I would have been ok with `c` as a var name ;)

Reply:
:smiley: addressed in https://github.com/cloudfoundry-incubator/quarks-utils/pull/42/commits/bc368dd626a3c70ba63a6bba5bd87198b3e4f510